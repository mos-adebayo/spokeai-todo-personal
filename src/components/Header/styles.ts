import styled from "styled-components";
import { gradientPrimaryBackground } from "../../util/stylesheet";

export const Wrapper = styled.div`
  display: flex;
  align-items: center;
  padding: 45px 20px;
  height: 100px;
  margin-bottom: 25px;
  font-family: "Mochiy Pop One", sans-serif;

  ${gradientPrimaryBackground};
`;
