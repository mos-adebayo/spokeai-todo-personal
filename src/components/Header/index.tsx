import React from "react";
import { Link } from "react-router-dom";
import { Wrapper } from "./styles";

const Header: React.FC = () => {
  return (
    <Wrapper>
      <Link to="/">
        <h3>Spoke.ai</h3>
      </Link>
    </Wrapper>
  );
};

export default Header;
