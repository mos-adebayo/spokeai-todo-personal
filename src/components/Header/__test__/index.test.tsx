import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import Header from "../index";

describe("<Header />", () => {
  let wrapper: ShallowWrapper;
  beforeEach(() => {
    wrapper = shallow(<Header />);
  });

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
