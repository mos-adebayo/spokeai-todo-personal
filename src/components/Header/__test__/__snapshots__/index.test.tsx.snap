// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match snapshot 1`] = `
<styled.div>
  <Link
    to="/"
  >
    <h3>
      Spoke.ai
    </h3>
  </Link>
</styled.div>
`;
